app/models/manageiq/providers/openstack/storage_manager/swift_manager/cloud_object_store_container.rb

```diff
@@ -31,6 +34,8 @@ def self.params_for_create(ems)
   end
 
   def self.raw_cloud_object_store_container_create(ext_management_system, options)
+    cloud_tenant_id = options.delete(:cloud_tenant_id)
```

Comment:
We should make sure these options are symbols since typically these will be strings when they are passed down through the API

Reply:
translation: we want to use that `symbolize_keys` when in the api when we call into here

Reply:
@agrare @kbrock Just updated the api pr to ensure they are being passed in as symbols.